src/content/en/2022/jamstack.md

```diff
@@ -4,10 +4,12 @@ title: Jamstack
 #TODO - Review and update chapter description
 description: Jamstack chapter of the 2022 Web Almanac covers adoption of technology, performance of websites built with Jamstack and weights for various resources.
 authors: [seldo, whitep4nth3r]
-reviewers: [GregBrimble, Nutlope]
-analysts: [seldo, GregBrimble]
-editors: []
+reviewers: [GregBrimble, Nutlope, tunetheweb]
+analysts: [seldo, tunetheweb]
+editors: [DesignrKnight]
 translators: []
+seldo_bio: Laurie has been a web developer since 1996, with occasional breaks to found companies like <a hreflang="en" href="https://www.crunchbase.com/organization/snowball-factory">awe.sm</a> (2010) and <a hreflanh="en" href="https://seldo.com/about/#:~:text=been%20a%20web,learn%20so%20much.">npm</a> (2014). He likes making websites, and he likes making the web bigger and better. He thinks one of the best ways to do that is to encourage more people to do web development, by teaching them existing techniques and by building tools and services that make web development easier, so they don't have to learn so much.
```

Comment:
@seldo, I pulled this from your website, but feel free to change as you see fit. This is listed at the bottom of the chapter.